roles/config-rh-sso/vars/main.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
move this to `default/main.yml` instead of using `vars`. Also, please don't use `default` in the var names as it won't make sense in any inventories that overrides the default value.

Reply:
Oh, thanks for pointing that out. This was an initial draft, I am not using those variables that way.
Fixed.